qiskit/visualization/circuit/_utils.py

```diff
@@ -197,22 +196,19 @@ def get_bit_register(circuit, bit):
     return bit_loc.registers[0][0] if bit_loc.registers else None
 
 
-@deprecate_arg("reverse_bits", since="0.22.0", package_name="qiskit-terra")
-def get_bit_reg_index(circuit, bit, reverse_bits=None):
+def get_bit_reg_index(circuit, bit):
```

Comment:
These deprecations can stay in place through 0.46 - we'll remove everything in one fell swoop in 1.0 instead, because we're trying to keep 0.46 as close to being "0.45 with more warnings" as possible.

Since we're looking at them, it'd be good to include a `removal_timeline="in the Qiskit 1.0 release"` on all the `deprecate_arg`/`deprecate_func` calls in this PR.